kivy/uix/scrollview.py

```diff
@@ -758,10 +758,24 @@ def on_scroll_move(self, touch):
             self._touch = False
             return self.on_scroll_start(touch, False)
         ud = touch.ud[uid]
-        mode = ud['mode']
```

Comment:
Sorry, I have a custom renamed widget and I edited this part multiple times. Obviously I missed this.